src/main/java/appeng/api/storage/IMEInventoryHandler.java

```diff
@@ -84,4 +84,13 @@ default boolean getSticky() {
         return false;
     }
 
+    /**
+     * Gets whether an inventory stores items for crafting. For example the {@link appeng.me.cache.CraftingGridCache}
+     *
+     * @return true if this inventory stores items for crafting
+     */
+    default boolean getCraftingInventory() {
```

Comment:
```suggestion
    default boolean isAutoCraftingInventory() {
```
Should start with `is` and adding "Auto" explains a bit better for what this means. Also rename the field.